### PR TITLE
Clone New York Times article page

### PR DESCRIPTION
**Pull request by Blaise-Shyaka**
In this project we did the following tasks:

- Create a fixed Navbar
- Create a hero section
- Embed a Youtube Video
- Add the article content 
- A grid with images and their caption
- Add sidebar floating to the right
- Add a fixed footer

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,879 @@
...
+
+  <body>
+    <div class="wrapper">
+      <nav class="nav-bar">
+        <div class="icons">
+          <a href="#"><i class="fa fa-bars" aria-hidden="true"></i></a>
+          <a href="#"><i class="fa fa-search" aria-hidden="true"></i></a>
+        </div>
+        <div class="logo">
+          <img src="./images/logo.png" alt="" />
```
Comment (expjazz) [inline index.html]: - [x] You are importing the logo with the wrong path, and it is not showing. The image in the project folder has a different extension (the PNG is in upper case)

File: README.md [hunk 2]
```diff
@@ -1 +1,47 @@
...
+- VISUAL STUDIO CODE
+- GIT
+- Lynters
+
+## Live Demo
+
+[Live Demo Link](https://raw.githack.com/rgcavalheiro/New-York-Times-Clone/nyt-article-page/index.html)
+
+
+
```
Comment (expjazz) [inline README.md]: - [x] You should add a section on how to open the project locally. It is a good practice to do this from here onwards.

**Review by expjazz — CHANGES_REQUESTED** [review]
## Status: Changes required :recycle: 

#### The page looks awesome, and your code is really clean and organized. You did a great job :100: 

However, there is one mistake with the logo in the navbar. Check the inline comments below for more detail. I also added one more idea so you can improve your readme file.

### Please make the required changes and resubmit for a new code review once you are ready
Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,706 @@
...
+  background-color: #eee;
+  border: 1px #eee solid;
+  padding: 5px;
+  text-align: center;
+  border-radius: 50%;
+}
+
+.fix i {
+  height: 10px;
+  width: 23px !important;
```
Comment (tirthajyoti-ghosh) [inline css/style.css]: Remove `!important`. Use higher specificity to override styles.

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,706 @@
...
+  align-items: center;
+}
+
+@media (min-width: 1200px) {
+  .picks {
+    margin: 15px 0;
+  }
+
+  .inflation-theory {
+    height: 800px !important;
```
Comment (tirthajyoti-ghosh) [inline css/style.css]: Remove `!important`. Use higher specificity to override styles.

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,706 @@
...
+  bottom: 19px;
+}
+
+.expand p {
+  margin: 0;
+}
+
+.fa-angle-up {
+  font-size: 20px !important;
+  font-weight: 900 !important;
```
Comment (tirthajyoti-ghosh) [inline css/style.css]: Remove `!important`. Use higher specificity to override styles.

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Changes required 🛑 

## Impressive progress so far 💪! Thanks for making all the changes 👏.

### However, we need to work on one change before we can merge.

- There is one CSS [bad practice](https://speckyboy.com/good-bad-css-practices) - no `!important` overusing, in your project. Use higher specificity to override styles.

Your project looks absolutely fantastic. Just make this change and request for another code review.

Happy coding 👨‍💻 

<hr>

**Tirthajyoti Ghosh**

[GitHub](https://github.com/tirthajyoti-ghosh) | [LinkedIn](https://www.linkedin.com/in/tirthajyoti-ghosh/) | [Twitter](https://twitter.com/terrific_ghosh)

**Review by expjazz — CHANGES_REQUESTED** [review]
## Status: Changes required :recycle: 

#### Like my colleague stated above, the page looks fantastic. 

However, you have linter errors and now there is a new bug on the page. It was perfect in the first review, so it must be something related to the styles you changed.

- [ ] 
image

### Please make the required changes and resubmit for a new code review once you are ready
Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,706 @@
...
+  color: #e2e2e2;
+}
+
+.picks {
+  display: flex;
+
+  /* border: 1px solid green; */
+  justify-content: space-around;
+  align-items: center;
+}
```
Comment (bertil291utn) [inline css/style.css]: ## Inline comments

Comments: It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /\* `.nav { color: red;}` \*/  
:heavy_check_mark: **comments about code** /\*Navigation \*/

- [x]  Check all your inline comments in the entire page and remove them

File: css/style.css [hunk 7]
```diff
@@ -0,0 +1,706 @@
...
+  z-index: 1;
+}
+
+.bottompart a:hover {
+  text-decoration: none;
+}
+
+.social-buttons {
+  display: flex;
+  justify-content: space-between;
```
Comment (bertil291utn) [inline css/style.css]: ## Using the Same Rule Repetitively[1]
- [x] Create a class once and the use in each HTML tag where you need it.
- [x] Delete all your `display:flex;` properties in entire CSS file
- [x] Add your HTML tags

**Example:**

**CSS file**

```
.flex{
  display:flex;
}
```
**Html file**

```
<div class="flex">
```

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

File: css/style.css [hunk 8]
```diff
@@ -0,0 +1,706 @@
...
+}
+
+.bottompart a {
+  color: rgb(102, 102, 102);
+}
+
+.moreinspace a {
+  color: black;
+  text-decoration: none;
+}
```
Comment (bertil291utn) [inline css/style.css]: ## Using the Same Rule Repetitively[1]
- [x] Create a class once and the use in each HTML tag where you need it.
- [x] Delete all your `display:flex;` properties in entire CSS file
- [x] Add your HTML tags

**Example:**

**CSS file**

```
.color-black{
  color: black;
}
```
**Html file**

```
<div class="color-black">
```

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

File: css/style.css [hunk 9]
```diff
@@ -0,0 +1,706 @@
...
+  justify-content: space-between;
+}
+
+.read button {
+  width: 350px;
+  padding: 10px;
+  color: white;
+  background-color: rgb(86, 123, 149);
+  border: none;
+  border-radius: 3px;
```
Comment (bertil291utn) [inline css/style.css:419]: ## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `display:flex;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.color-white{
  color: white;
}
```
**Html file**

```
<div class="color-white">
```

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

**Review by bertil291utn — CHANGES_REQUESTED** [review]
**Status:** Changes requested 🟠  :shipit:? Not yet.

# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.

### Your work is amazing you master it GitHub Flow you have a professional README file. 

However, in order to accomplish as the actual page, you required to do these changes, listed below:

- Improve your README file
  - [x]  Remove your image and add a new big one nad should just the main page not entire web page

### Navigation bar

- [x] The NY Times logo is a link ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))

### Main content

- [x] Add a margin o remove additional grid rows you have in the theory of inflation section

image

##### Bottom part

* [x] It contains links to other articles with image previews on the leftmost side ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))

* [x] It contains links to more popular articles on the rightmost side ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))

image

### Summary

- Please also check inline comments below this comment box

<hr>

Guys work in the requested changes above and request another review.
Good luck. 

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

File: css/style.css (hunk 9 above)
Comment (rgcavalheiro) [reply css/style.css:419]: can you review this please?

**Comment by bertil291utn** [comment]
**Status:** Changes requested 🟠  :shipit:? Not yet.

# Hey @Blaise-Shyaka  and @rgcavalheiro , you've been doing a great job

### Your work is amazing all your changes have been approved

However, in order to accomplish as the actual page, you required to do these changes, listed below:

- [x] When you have links change color as the original page 
- [x] The icons in the navbar and the footer should be black and the text should be this color `#326891;`

image

### Summary

- In order to accomplish as the original one please make the requested changes 

<hr>

Guys work in the requested changes above and request another review.
Good luck. 

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

**Comment by bertil291utn** [comment]
**Status:** Approved ✅

# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.



<h2 align="center">Ship it </h2>

<p align="center">image</p>

Please merge to master branch and move on with the next assignment.

You're doing  a great job keep going 💻 🚀 🔥

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

**Pull request merged**